emission/analysis/modelling/user_label_model/model_storage.py

```diff
@@ -0,0 +1,129 @@
+from enum import Enum
+from typing import Dict, Optional
+from emission.analysis.modelling.user_label_model.model_type import ModelType
+import emission.core.wrapper.user_label_prediction_model as ecwu
+import emission.storage.pipeline_queries as epq
+import emission.storage.timeseries.abstract_timeseries as esta
+import emission.storage.decorations.analysis_timeseries_queries as esda
+import emission.analysis.modelling.user_label_model.util as util
+import logging
+import arrow
+import pymongo
+from emission.storage.timeseries.builtin_timeseries import BuiltinTimeSeries
+
+from emission.storage.timeseries.timequery import TimeQuery
+
+class ModelStorage(Enum):
+    FILE_SYSTEM = 0
+    DATABASE = 1
+    @classmethod
+    def names(cls):
+        return list(map(lambda e: e.name, list(cls)))
+
+
+def create_filename(user_id, model_type: ModelType) -> str:
+    return f"user_label_model_{model_type.name}_{str(user_id)}"
+
+
+def load(user_id, model_type: ModelType, model_storage: ModelStorage) -> Optional[Dict]:
+    """load a user label model from a model storage location
+
+    :param user_id: the user to request a model for
+    :param model_type: expected type of model stored
+    :param model_storage: storage format 
+    :return: the model representation as a Python Dict or None
+    :raises: TypeError if loaded model has different type than expected type
+    """
+    if model_storage == ModelStorage.FILE_SYSTEM:
+        filename = create_filename(user_id, model_type)
+        model_data = util.load_fs(filename)
+        return model_data
+    elif model_storage == ModelStorage.DATABASE:
+        
+        # retrieve stored model with timestamp that matches/exceeds the most
+        # recent PipelineState.USER_LABEL_MODEL entry        
+        ts = esda.get_timeseries_for_user(user_id)
+        if not isinstance(ts, BuiltinTimeSeries):
+            raise Exception('user model storage requires BuiltInTimeSeries')
```

Comment:
as discussed, we don't want to have the filesystem code around for the long term.
It is simply an artifact of the implementer not knowing how to work with the database.
We will always have a database to store the raw objects, so the file system support is unnecessary complexity.

You should remove the filesystem option and remove all the related config flags.
If you really want to support backwards compat, you should fallback to reading from the file in the one implementation.
https://github.com/e-mission/e-mission-server/pull/852#discussion_r895077533